### ISSUE TITLE
the pad_batch function is error?

### ISSUE THREAD

**Issue by quoniammm** [EdGENetworks/attention-networks-for-classification#8]
![image](https://user-images.githubusercontent.com/12003094/31044314-8df28e9e-a5ff-11e7-8a73-1d84f3bb34b9.png)

I feel that `np.max` should be used.However, here is `np.mean`

**Comment by Sandeep42** [comment]
Yeah, np.max should be used ideally. I used mean because it is consuming too much of GPU memory. Now, I think you should ideally sort samples with their lengths so that the padding could be minimised. That hasn't been implemented here. You can have a look here.

https://github.com/EdGENetworks/anuvada
